src/scaraplate/rollup.py

```diff
@@ -85,8 +85,8 @@ def rollup(
         cookiecutter_config = cookiecutter_config_path / "cookiecutterrc.yaml"
         cookiecutter_config.write_text(
             f"""
-cookiecutters_dir: "{cookiecutter_config_path / 'cookiecutters'}"
-replay_dir: "{cookiecutter_config_path / 'replay'}"
+cookiecutters_dir: "{(cookiecutter_config_path.resolve() / 'cookiecutters').as_posix()}"
+replay_dir: "{(cookiecutter_config_path.resolve() / 'replay').as_posix()}"
```

Comment:
`as_posix()` call indeed looks appropriate here, but is the `resolve()` call necessary? On Linux, it can expand symlinks (such as `/home/myuser` to a path like `/storage/home/myuser`), which might not be wanted here.

Reply:
On second thought, as this file is relevant only during the rollup operation, i.e. it's not stored, I suppose it's okay to expand symlinks here